src/Adapter/Presenter/Order/OrderLazyArray.php

```diff
@@ -179,6 +179,9 @@ public function getProducts()
 
             $productPrice = $includeTaxes ? 'product_price_wt' : 'product_price';
             $totalPrice = $includeTaxes ? 'total_wt' : 'total_price';
```

Comment:
```suggestion
            if (is_array($orderProduct['customizedDatas']) && count($orderProduct['customizedDatas'])) {
                $totalPrice = $includeTaxes ? 'total_customization_wt' : 'total_customization';
            } else {
                $totalPrice = $includeTaxes ? 'total_wt' : 'total_price';
            }
```

:thinking:  Don't you think it would be clearer to have if/else statement instead of default value overridden in some cases ?

Just a suggestion ^^

Reply:
Yes, I see it. In this case it's definitely more clearer. Thanks :+1: